common/network-types/src/session/protocol.rs

```diff
@@ -495,7 +495,9 @@ mod tests {
         assert_eq!(10, SessionMessage::<0>::SEGMENT_OVERHEAD);
         assert_eq!(8, SessionMessage::<0>::MAX_SEGMENTS_PER_FRAME);
 
-        assert!(SessionMessage::<0>::MAX_MESSAGE_SIZE < 2048);
+        const _: () = {
+            assert!(SessionMessage::<0>::MAX_MESSAGE_SIZE < 2048);
+        };
```

Comment:
@NumberFour8 you will love this. I found this deep inside a very special library.